syfertext/vectors.py

```diff
@@ -2,6 +2,7 @@
 import os
 from pathlib import Path
 import numpy as np
+import torch
 import importlib
 from typing import Union
 
```

Comment:
```suggestion
import syft as sy
hook = sy.TorchHook(torch)
```

Reply:
Why do we need the hooking here? It is already done in `__init__()`

Reply:
Oh, I did not consider that, my bad.

Reply:
:+1: